core/src/main/java/org/kiwix/kiwixmobile/core/main/TabsAdapter.java

```diff
@@ -145,6 +145,10 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
       listener.onSelectTab(v, selectedPosition);
       notifyDataSetChanged();
     });
+    if (!webViewTitle.equals(activity.getString(R.string.menu_home))) {
+      NightModeViewPainter painter = new NightModeViewPainter(holder.content.getContext());
```

Comment:
NightModeViewPainter should not be constructed by you, only Dagger

Reply:
@macgills how do I do that in the TabsAdapter? TabsAdapter isn't created by dagger.

Reply:
Pass it to the constructor of TabsAdapter